matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.css

```diff
@@ -449,4 +449,15 @@ label.score {
 }
 .comment-text {
   padding: 5px!important;
+}
+.phenotypes-div ul li {
+  margin: 2px 2px 2px 0;
+  padding: 0.5em;
+  display: inline-block;
+  line-height: 1.2em !important;
+}
+.matched {
+  background-color: #EEE;
+  vertical-align: top;
```

Comment:
This change had actually been proposed to go along with the display of bullets for `li`, but since we're not doing that anymore it's no longer needed.